dss/subscriptions_v2/__init__.py

```diff
@@ -16,11 +16,25 @@ class SubscriptionData:
     FORM_FIELDS = 'form_fields'
     PAYLOAD_FORM_FIELD = 'payload_form_field'
     ATTACHMENTS = 'attachments'
+    STATS = 'stats'
+    BLANK_STATS = {"attempted": 0, "successful": 0, "failed": 0}
 
 
 subscription_db_table = f"dss-subscriptions-v2-{{}}-{os.environ['DSS_DEPLOYMENT_STAGE']}"
 
 
+def update_subscription_stats(doc: dict, status: bool):
+    status_type = 'successful' if status else 'failed'
```

Comment:
Instead of `status: bool`, would it be easier to just have `status: str` and return `"failed"` or `"successful"` directly?